sdk-api/src/main/java/com/xee/auth/AuthenticationActivity.java

```diff
@@ -43,7 +43,7 @@
 /**
  * Xee Authentication SDK
  */
-public final class AuthenticationActivity extends AppCompatActivity {
```

Comment:
You're right. I just removed it. Please see 6c9e9038625b1b3f7c9d25da3c0f75c64cc746aa